portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -77,7 +77,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         String sender = (String) entity.getProperty("sender");
         String image = (String) entity.getProperty("imgUrl");
         ArrayList<String> messageReplies = (ArrayList) entity.getProperty("replies");
-        BlogMessage message = new BlogMessage(messageId, tags, comment, image, sender, messageReplies, timestamp);
+        BlogMessage message = new BlogMessage(messageId, tags, comment, image, sender, "", messageReplies, timestamp);
```

Comment:
A general tip: if we need to pass a constant (String, int, double, bool, etc.) to a function argument, it's usually good to use an inlined comment like

BlogMessage message = new BlogMessage(
    messageId, tags, comment, image, sender, /*email=*/"", messageReplies, timestamp);

Then reader knows what argument is getting the "" empty string.